Documentation/devel/stage1-implementors-guide.md

```diff
@@ -50,10 +50,10 @@ An alternative stage 1 could forego systemd-nspawn and systemd altogether, or re
 All that is required is an executable at the place indicated by the `coreos.com/rkt/stage1/run` entrypoint that knows how to apply the pod manifest and prepared ACI file-systems to good effect.
 
 The resolved entrypoint must inform rkt of its PID for the benefit of `rkt enter`.
-Stage 1 must write the host PIDs of the pod's process #1 and that process's parent to these two files, respectively:
+Stage 1 implementors have two options for doing so; only one must be implemented:
```

Comment:
We should clarify in the code [here](https://github.com/coreos/rkt/blob/19bef69d75169dad1591c5d59c2f94ddbf8a44a0/rkt/pods.go#L918) that the ordering is not important and only one of them must be supplied (currently the code reads as if it is favouring "pid", but if you factor in the [raciness](https://github.com/coreos/rkt/blob/19bef69d75169dad1591c5d59c2f94ddbf8a44a0/rkt/pods.go#L937) of the function then we can consider that ordering to be arbitrary).